github/issues_comments.go

```diff
@@ -10,6 +10,9 @@ import (
 	"time"
 )
 
+// MaxCommentLen is the maximum length of a comment that github will accept without a validationg error.
+var MaxCommentLen = 65535
```

Comment:
If GitHub ever changed their limit, then we should simply update the library.  I think I'd rather leave this unexported for the time being.  We could always expose it later if there is a strong argument for it, but it's much harder to go the other way.